conans/test/functional/compression/compression_test.py

```diff
@@ -0,0 +1,443 @@
+import abc
+import ctypes
+import os
+import random
+import six
+import stat
+import string
+import sys
+import tarfile
+import time
+import uuid
+import zipfile
+
+import unittest
+from nose.plugins.attrib import attr
+from parameterized import parameterized_class
+
+from conans.client.tools.files import chdir, load
+from conans.util.files import md5sum
+from conans.test.utils.tools import TestClient
+
+
+# more portable implementation, works on Windows 10 with developer mode (non-admin symbolic links)
+def create_symlink(src, dst):
+    # on Windows, special treatment is needed
+    if os.name == 'nt':
+        flags = 2  # SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE
+        if os.path.isdir(src):
+            flags |= 1  # SYMBOLIC_LINK_FLAG_DIRECTORY
+        CreateSymbolicLinkW = ctypes.windll.kernel32.CreateSymbolicLinkW
```

Comment:
I am fine if we don't manage symlinks in Windows, we have never done it, and haven't got a single complain or report.

Reply:
I personally think this is kinda important, as we usually extract the same archive on different operation systems, and we have to ensure it works correctly in all the cases, e.g. symbolic links are handled gracefully cross-platform.
we may not encounter an issues yet, but will definitely have in feature (Windows 10 added non-admin symlink is still pretty recent, but more and more users will start to utilize that).

Reply:
we at least had https://github.com/conan-io/conan/issues/5774
maybe more issues, I'll try to find more and link here

Reply:
checked once again, and it seems like everything works for me with `os.path.islink` and `os.symlink`, so I have removed these Windows-specific hacks